shared/middleware/helpers/core.rb

```diff
@@ -46,9 +46,19 @@ def not_found()
 end
 
 def too_large()
-  halt(413, "Payload too large\n")
```

Comment:
The only caller right now is in files_api.rb. I suspect that if I went this approach I'd want to change the other nearby methods to behave similarly. Maybe I'm better off just leaving this alone, having my TooLarge class live in property_bag, and just live with the fact that we have two different ways of responding to requests with 413?